Makefile

```diff
@@ -140,8 +142,8 @@ test-all-valgrind: test-build
 	$(PYTHON) tools/test.py --mode=debug,release --valgrind
 
 test-ci: | build-addons
-	$(PYTHON) tools/test.py -p tap --logfile test.tap --mode=release \
-		addons message parallel sequential
+	$(PYTHON) tools/test.py $(TEST_CI_ARGS) -p tap --logfile test.tap --mode=release \
+    --flaky-tests=$(FLAKY_TESTS) addons message internet parallel sequential
```

Comment:
@thefourtheye what do you think would be the correct indentation? I am not sure what the rule is for makefiles :smile: 
The behavior is set by the different jobs based on their configuration. The jobs expect the default to be `run`